mirdata/datasets/saraga_carnatic.py

```diff
@@ -289,8 +290,13 @@ def load_tonic(tonic_path):
     if not os.path.exists(tonic_path):
         raise IOError("tonic_path {} does not exist".format(tonic_path))
 
-    with open(tonic_path, 'r') as reader:
-        return float(reader.readline().split('\n')[0])
+    tonic = 0
```

Comment:
why do you need `tonic=0`?

Reply:
True. I was initializing the variable but I guess it's not needed here. Sorry!